src/main/java/com/code_intelligence/jazzer/mutation/mutator/proto/BuilderMutatorFactory.java

```diff
@@ -93,8 +102,27 @@ private static <T extends Builder, U> InPlaceMutator<T> mutatorForField(
         return mutateViaView(
             builder -> makeMutableRepeatedMessageFieldView(builder, field), underlyingMutator);
       } else {
-        InPlaceMutator<List<U>> underlyingMutator =
-            (InPlaceMutator<List<U>>) factory.createInPlaceOrThrow(typeToMutate);
+        InPlaceMutator<List<U>> underlyingMutator;
+        if (field.getJavaType() == JavaType.ENUM) {
+          // Proto enum fields are special as their type (EnumValueDescriptor) does not encode their
+          // domain. We deal with this by using a synthetic factory for the list element mutator
+          // that associates a concrete EnumDescriptor with this type.
+          SerializingMutator<EnumValueDescriptor> elementMutator = mutatorForEnum(field);
+          underlyingMutator =
+              (InPlaceMutator<List<U>>) factory
+                  .tryCreate(new TypeHolder<List<EnumValueDescriptor>>() {}.annotatedType(),
+                      new MutatorFactory() {
```

Comment:
Ha! Using an anonymous `MutatorFactory` is cheating! :smile:

I couldn't figure out how to create a `List` mutator for a type with a "dynamic mutator". This is quite ingenious.